core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -676,10 +675,12 @@ Result remove(K key, V oldValue, OakTransformer<V> transformer) {
             if (!ctx.isKeyValid()) {
                 // There is no such key. If we did logical deletion and someone else did the physical deletion,
                 // then the old value is saved in v. Otherwise v is (correctly) null
-                return transformer == null ? ctx.result.withFlag(logicallyDeleted ? TRUE : FALSE) : ctx.result.withValue(v);
+                return transformer == null ? ctx.result.withFlag(logicallyDeleted ? ValueUtils.ValueResult.TRUE :
```

Comment:
Here and below, hard to understand what is compared with what and what is returned. Please put some parenthesis

Reply:
I don't see how parenthesis would help.
The only change is that now TRUE and FALSE are addressed directly instead of via a static import.
It easier to read because you can immediately know where those TRUE/FALSE values come from.